lib/r10k/cli/deploy.rb

```diff
@@ -53,7 +53,7 @@ def self.command
           DESCRIPTION
 
           flag :p, :puppetfile, 'Deploy modules from a puppetfile'
-          required :d, :default_branch_override, 'Specify a branchname to override the default branch in the puppetfile'
+          required :default-branch-override, 'Specify a branchname to override the default branch in the puppetfile'
```

Comment:
Should this be:
```
required nil, :'default-branch-override', 'Specify ...'
```
Just guessing based on other examples in the file that something is missing here.

What's the difference between `required` and `flag`? Guessing it's non-intuitive but as long as I'm commenting thought I'd ask. :)

Reply:
Good catch! Thank you.

As for the difference between 'flag' and 'required':

`flag`: indicates a true/false value
`required` implies that a _parameter_ is required (which is counter-intuitive, and apparently has [since been deprecated in the library we use](https://github.com/ddfreyne/cri#command-line-options)).